binding.gyp

```diff
@@ -26,15 +26,20 @@
                 "<!(node -e \"require('nan')\")"
             ],
             "cflags!": [ "-fno-exceptions", "-fno-rtti" ],
-            "cflags_cc!": [ "-fno-exceptions", "-fno-rtti"  ],
-            "cflags_cc": [ "-std=c++17" ],
+            "cflags_cc!": [ "-fno-exceptions", "-fno-rtti", "-std=gnu++20"  ],
+            "cflags_cc": [ "-std=gnu++2a" ],
```

Comment:
Shouldn't this change also apply to [CMakeList.txt](https://github.com/DataDog/dd-native-iast-taint-tracking-js/blob/main/CMakeLists.txt#L3C6-L3C24)?

Reply:
Changed